datafusion/sqllogictest/test_files/aggregate.slt

```diff
@@ -173,9 +173,6 @@ SELECT approx_percentile_cont(c12) WITHIN GROUP (ORDER BY c12) FROM aggregate_te
 statement error DataFusion error: This feature is not implemented: Tdigest max_size value for 'APPROX_PERCENTILE_CONT' must be a literal
 SELECT approx_percentile_cont(0.95, c5) WITHIN GROUP (ORDER BY c12) FROM aggregate_test_100
 
-statement error DataFusion error: This feature is not implemented: Conflicting ordering requirements in aggregate functions is not supported
```

Comment:
Why was this test removed? I think it is still a valid test isn't it?

Reply:
Actually not -- this query works now. I looked at what is going on and it seems like the requirements are not really conflicting, seems like the old equivalence system wasn't able to handle it.

It'd be great if you can double check as well, so we don't accidentally start accepting queries we can't currently execute correctly :)